src/pages/studyView/UserSelections.tsx

```diff
@@ -809,6 +828,60 @@ export default class UserSelections extends React.Component<
         });
     }
 
+    private groupedStructVarQueries(
```

Comment:
comment here explaining what "grouped" means.  seems slighly odd because i would expect this function to do some grouping, but it seems just a map from query to pilltag.

Reply:
I am adhering to the naming convention in the UserSelections.ts file. I think that this discussion is out of scope for this PR.